static/app/views/releases/list/releaseListInner.tsx

```diff
@@ -130,6 +134,8 @@ export default function ReleaseListInner({
   );
 }
 
+export default withSentryRouter(ReleaseListInner);
```

Comment:
could be better to call `const router = useRouter();` inside the component, and drop the `router: InjectedRouter` at the same time instead of adding an HoC. simpler to replace the useRouter call later.